MANIFEST.in

```diff
@@ -1,2 +1,3 @@
 recursive-include KDEpy *.c
+recursive-include KDEpy *.pyx
```

Comment:
Is this related to the PR? If so, how? 
Same question applies to the setup.py change.